### ISSUE TITLE
Check if value exists, than return it, else return something else in ReactiveList

### ISSUE THREAD

**Issue by Danelicious** [appbaseio/reactivesearch#972]
**Issue Type:**
Question

**Platform:**
ReactiveSearch for Web 

**Description:**
I have items in ES with articles that consist of a title, description and a link. But sometimes there is only the title and description, no link. 

I would like to check if there is a link in my ES-index. If yes, I want to return the link in my ReactiveList. If not, I want to return "No link available". I tried a couple of things but failed. 

Should be something like this, right?

`function LinkChecker({value}) {
  if (!value) {
    return <div>No link available.</div>;
  } else {
    return (
      <div>
        {res.url}
      </div>
    );
  }
}
`  

So, basically I want to use something like the renderNoResults prop but only when a value is null whereas others still exist and get displayed.

**Comment by Danelicious** [comment]
I finally got it! Of course it has nothing to do with ReactiveSearch, so I will close it. Feel free to delete the post. Cheers!
